### ISSUE TITLE
Unexpected error thrown when trying to get access to the keyboardType in TextField

### ISSUE THREAD

**Issue by jgprogram** [nalexn/ViewInspector#171]
**Environment:**

- ViewInspector 0.9.1
- Xcode 13.2.1
- iOS Deployment Target 14.0
- Swift 5

**Code to reproduce:**

```Swift
import XCTest
import ViewInspector
import SwiftUI

struct SampleView: View {
    @State
    var text = "Any text"

    var body: some View {
        VStack {
            TextField("", text: $text)
                .keyboardType(.alphabet)
        }
    }
}

class SampleViewSpec: XCTestCase {

    func test() throws {
        let view = try! SampleView().body.inspect()
        try XCTAssertEqual(view.vStack().textField(0).keyboardType(), UIKeyboardType.alphabet)
    }
}
```

**Error thrown:**

`unexpected error thrown: <Type mismatch: WritableKeyPath<EnvironmentValues, KeyboardType> is not WritableKeyPath<EnvironmentValues, Int>>`

**Description:**
When i am trying to assert keyboard type from text field I am getting the above error. I checked it on 2 independent computers.

**Possible fix**
In file TextInputModifiers adjust line 18 

Current
```Swift
let keyPath = try Inspector.environmentKeyPath(Int.self, reference)
```

Proposal
```Swift
let keyPath = try Inspector.environmentKeyPath(KeyboardType.self, reference)
```

**Comment by toto1010** [comment]
I have exact same issue!

**Comment by gwrn** [comment]
I think I also had this issue in of my projects but I'm not 100% sure since this whole iOS development is one big struggle with bugs.

**Comment by nalexn** [comment]
Thanks for the report! This is fixed in the coming next version.

**Comment by nalexn** [comment]
Released with v0.9.2
